src/ast/rewriter/bit_blaster/bit_blaster_adder.cpp

```diff
@@ -0,0 +1,112 @@
+#include "ast/rewriter/bit_blaster/bit_blaster_adder.h"
```

Comment:
Even as the Contribution license already covers it some OSS users ask for copyright in all files. 
Would be useful to also include acknowledgement of you as author.